src/components/selector-dropdown/SelectorDropdown.scss

```diff
@@ -4,14 +4,34 @@
  * Selector Dropdown
  **************************************/
 
-.selector-dropdown-wrapper {
+.SelectorDropdown {
     position: relative;
     z-index: 3; /* increase z-index based on max-height of dropdown wrapper */
 
     .overlay-wrapper,
     .overlay {
         display: block;
         width: 100%;
+
+        .SelectorDropdown-divider {
+            border: 0;
+            border-bottom: 1px solid $bfive;
+            display: block;
+            height: 1px;
+            margin: 5px 5%;
```

Comment:
Does the left-right margin need to be a percentage? Could we use the 15px like in the title?

Reply:
It doesn't need to be a percentage. I'll update this.